### PR TITLE
Updating Steering Committee membership and include information about when their terms expire.

### PR DESCRIPTION
**Pull request by pstout**

### CODE REVIEW

File: governance.md [hunk 1]
```diff
@@ -108,9 +108,11 @@ Duties:
 Qualifications:
 
 * Nominated and Appointed by Steering Committee
-  * 2 year terms, term limit of consecutive 4 years (after, individual must take off at least 1 year)
+  * 2 year terms, term limit of 4 years consecutively^ (after which, an individual must take off at least 1 year)
   * Terms should be staggered, which should result in approximately half of the committee up for election each year
-  * Term starts w/calendar year (i.e. January 1st)
+  * Terms run from February 1st to January 31st two years later
+
+^ - SC members whose terms where extended by one month when the terms where shifted from starting January 1st to starting February 1st may serve 4 years and one month consecutively.
```
Comment (duftler) [inline governance.md:115]: s/where/were

**Review by duftler — APPROVED** [review]

File: governance.md [hunk 2]
```diff
@@ -119,11 +121,19 @@ Logistics
...
-* Peter Stout (Netflix)
-* Isaac Mosquera (Armory)
-* Andrew Philips (Google)
+* Andy Glover (Netflix) - Chair - Term expires 2022.01.31
+* Matt Duftler (Google) - Term expires 2022.01.31
+* Peter Stout (Netflix) - Term expires 2021.01.31
+* Isaac Mosquera (Armory) - Term expires 2022.01.31
+* Andrew Philips (Google) - Term expires 2021.01.31
+
+SC members as of 2021.02.01:
```
Comment (plumpy) [inline governance.md:130]: I think you can delete this part now?

**Review by plumpy — APPROVED** [review]

File: governance.md (hunk 2 above)
Comment (pstout) [reply governance.md:130]: Oops, yes.

File: governance.md (hunk 1 above)
Comment (pstout) [reply governance.md]: Fixed

File: governance.md (hunk 2 above)
Comment (pstout) [reply governance.md]: Fixed

**Pull request merged**